packages/app/background-jobs-common/UPGRADING.md

```diff
@@ -0,0 +1,45 @@
+# Upgrading Guide
+
+## Upgrading from `4.0.0` to `5.0.0`
+
+### Description of Breaking Changes
+
+1. **Property Removal**
+  - The deprecated `processCalls` property from `FakeBackgroundJobProcessor` has been removed.
+
+2. **Method Changes in `AbstractBackgroundJobProcessor`**
+  - The method `getActiveQueueIds` has been removed. A new constant method now serves this purpose.
+  - `logJobStarted` and `logJobFinished` methods are no longer public. The SDK now offers basic logging, and there are
```

Comment:
can we mention these alternative ways?

Reply:
Added a link to the migration steps section where it is described